src/service/FileTransferService.ts

```diff
@@ -42,10 +47,62 @@ export class FileTransferService {
                 }
             }).catch((err) => {
                 if (err.code === UNSUPPORTED_MEDIA_TYPE) {
-                    throw new Error('Unsupported file type')
+                    throw new Error('Unsupported file type');
                 } else {
-                    throw new Error(err.message)
+                    throw new Error(err.message);
                 }
             });
     }
+
+    async fileMetada(fileId: string): Promise<FileMetada> {
+        const config: AxiosRequestConfig = {
+            headers: {
+                'x-api-key': this.key
+            },
+        };
+
+        return await axios
+            .get<FileMetada>(`${this.url}/${fileId}`, config)
+            .then(_ => _.data)
```

Comment:
May I suggest we stop using `_` as names? There was time I liked that approach especially working with other programming languages but at the end of the day names matters and `_` is against good naming conventions.